src/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -231,6 +231,7 @@ public sealed partial class MethodBuilder : System.Reflection.MethodInfo
         public bool InitLocals { get { throw null; } set { } }
         public override bool IsGenericMethod { get { throw null; } }
         public override bool IsGenericMethodDefinition { get { throw null; } }
+        public override bool IsConstructedGenericMethod { get { throw null; } }
```

Comment:
It should be implemented in the source (CoreCLR).

This method has a default implementation which should suffice so you might have to do anything here, unless one of our api compat checking tools complain about an override method missing in the implementation.